src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/FunctionId.cs

```diff
@@ -548,5 +548,10 @@ internal enum FunctionId
         Refactoring_FixAllOccurrencesPreviewChanges = 563,
 
         LSP_UsedForkedSolution = 571,
+
+        DocumentOutline_WindowOpen = 580,
+        DocumentOutline_SortByName = 581,
```

Comment:
I don't think we use any of these other than `DocumentOutline_WindowOpen`? Should we or can we remove this additional enums?

Reply:
Oh they're used in `DocumentOutlineHelper.cs` in the `Sort()` method.